index.html

```diff
@@ -144,7 +144,22 @@
               Smuggler's Copter
             </a>
           </li>
+          <li>
+            <span class="icon tip" title="Aether Revolt"><i class="ss ss-aer"></i></span>
+             <a target="_blank" href="http://gatherer.wizards.com/Pages/Card/Details.aspx?name=Felidar+Guardian">Felidar Guardian</a>
+          </li>
         </ul>
+
+        <div class="ban-source-toggle-container">
+          <a id="ban-source-toggler" href="#" class="low-distraction" onclick="toggleBanSources();">
+            show announcements
```

Comment:
Oh, awesome! I was just about to take a look. Wound up playing a lot of AKH yesterday. 😉